File: src/main/scala/ch2/el/DemoService.scala
package ch2.el

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Service

import scala.beans.BeanProperty

/**
  * Created by mike on 2016/10/18.
  */
@Service
class DemoService {
  @Value("properties of other Class")
  @BeanProperty
  var another:String = null
}

File: base_exercise/src/main/java/springboot/chapter2/el/DemoService.java
package springboot.chapter2.el;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Service;

/**
 * Created by mike on 2016/10/26.
 */
@Service
public class DemoService {

    @Value("I am come from DemoService")
    private String another;

    public String getAnother() {
        return another;
    }

    public void setAnother(String another) {
        this.another = another;
    }
}
